spec/v1/spec.md

```diff
@@ -527,8 +522,14 @@ There may be different motivations for interacting with [[ref:Trust Establishmen
 
 ### Topic Registry
 
-::: todo Topic Registry
-  Create a place for topics to be registered.
+As per the section on [Publication & Discovery](#publication-discovery) there are a number of possibilities for hosting [[ref:Trust Establishment Documents]]. A related concept is defining a set of well-known [[ref:Topics]] that can find utility across an ecosystem of trust. One such option in housing these well-defined [[ref:Topics]] is a _Topic Registry_. Topic Registries can be as simple as [GitHub repositories](https://github.com/) that host a collection of topics or a website in a similar fashion to [Schema.org](https://schema.org/).
```

Comment:
@csuwildcat let me know what you think about this section, specifically on the accuracy of the DWN text